splines/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ if(NOT CPU_ONLY)
     target_compile_definitions(Splines PUBLIC CUDA_ERROR_CHECK)
   endif()
   set_property(TARGET gpuSpline Splines PROPERTY CUDA_ARCHITECTURES 35 52 60 61 70 75 80 86)
-  target_link_libraries(gpuSpline)
+  target_link_libraries(gpuSpline cuda-samples)
 else()
```

Comment:
I think this might need tidy
https://github.com/mach3-software/MaCh3/blob/212bc3d419015b54c8684b98ceb2b439913dbc4f/cmake/Modules/CUDASetup.cmake#L6

Reply:
Ah I hadn't seen that